Leanplum-SDK/Classes/LPActionManager.h

```diff
@@ -50,16 +50,6 @@ typedef enum {
 #define  LP_HELD_BACK_ACTION @"__held_back"
 
 @interface LPActionManager : NSObject {
-  @private
```

Comment:
You moved a few interfaces defined in the header class into the implementation class, it seems like your reasoning was because these were private interfaces anyway; however are we sure that there aren't any other classes importing these header files and that this change of visibility wouldnt have an adverse outcome?

Reply:
even if they're imported, they're private, and hence inaccessible to any other classes